### PR TITLE
Framework for Guest mode

### PR DESCRIPTION
**Pull request by kennethlien**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

Starting Guest mode, someone on Airtable is now named Guest and has a phone number of ##########.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Online sources

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review

Try breaking it by messing w/ signing up and logging in, Airtable, etc.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps

Restyle, probably shouldn't just use edited empty states.

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots

Simulator Screen Shot - iPhone 11 - 2020-04-02 at 19 26 54
Simulator Screen Shot - iPhone 11 - 2020-04-02 at 19 27 11
Simulator Screen Shot - iPhone 11 - 2020-04-02 at 19 27 15

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @anniero98 @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: screens/rewards/RewardsScreen.js [hunk 1]
```diff
@@ -33,12 +33,15 @@ export default class RewardsScreen extends React.Component {
...
       index: tab,
       routes,
+      isGuest: false,
     };
   }
 
   // Load user record & transactions
   async componentDidMount() {
     const userId = await AsyncStorage.getItem('userId');
+    const isGuest = userId === 'recxEGfvExP4Dv8nr';
```
Comment (wangannie) [inline screens/rewards/RewardsScreen.js:43]: Will eventually maybe have to change this to depend on dev or prod environment but I guess that's a later problem : ) @anniero98
